can-simple-observable.js

```diff
@@ -56,13 +58,25 @@ SimpleObservable.prototype = {
 		queues.enqueueByQueue(this.handlers.getNode([]), this, [value, old], function(){
 			return {};
 		});
+		//!steal-remove-start
+		if (this._log) {
+			this._log(old, value);
+		}
+		//!steal-remove-end
 	},
 	// .on( handler(newValue,oldValue), queue="mutate")
 	on: function(handler, queue){
 		this.handlers.add([queue|| "mutate", handler]);
 	},
 	off: function(handler, queue){
 		this.handlers.delete([queue|| "mutate", handler]);
+	},
+	log: function() {
+		//!steal-remove-start
+		this._log = function(previous, current) {
+			dev.log("SimpleObservable<" + current + "> (was <" + previous + ">)");
```

Comment:
we should probably use `dev.log("SimpleObservable<> is ",current," was ", previous)`

make sure we log stuff people can explore and click on, don't assume it's always a string.

Reply:
I just pushed some improvements:

1) The observable name is no longer hardcoded but `canReflect.getName` is used instead. This gives the change to mutate it before the log is called

```js
var obs = ...
canReflect.setName(obs, "AgeObservable");
obs.log();
```

2) I noticed that strings do not include quotes by default in the terminal and printing the actual value to the terminal in a single line makes easy to miss the previous value.

See that `1` is actually an string
![screen shot 2017-10-16 at 17 46 56](https://user-images.githubusercontent.com/724877/31640534-58248156-b29c-11e7-9a53-799d03fe627b.png)

see how 'was "one"' is pushed because of the current object.
![screen shot 2017-10-16 at 17 43 10](https://user-images.githubusercontent.com/724877/31640559-6c852722-b29c-11e7-98b3-6a6153998052.png)

I made it so each thing is its own line and there is a helper to make strings easy to identify in the output:

![screen shot 2017-10-16 at 17 46 07](https://user-images.githubusercontent.com/724877/31640599-9dbaf8c6-b29c-11e7-883e-ad4502de939c.png)

It looks nice in Firefox, too.
![screen shot 2017-10-16 at 17 49 15](https://user-images.githubusercontent.com/724877/31640608-adf1ae60-b29c-11e7-801f-5ef271f5e1bd.png)